python/enthic/app.py

```diff
@@ -178,23 +178,83 @@ def pre_cast_integer(probe):
     return str(probe) if probe.__class__ is int else probe if probe.isnumeric() is True else None
 
 
-def result_array(probe, limit, offset=0):
+
+def ape_converting(ape_code):
+    """
+    APE code are not save in the original format.
+    This function returns the corresponding code in MySQL or None
+    if the code does not exist
+
+        :param ape_code : APE_CODE filter
+        :param APE_CODE : all known APE code
+
+        :return: list of corresponding  APE_CODE in MySQL
+    """
+    #import re
+    #ape = list()
+    for i in APE_CODE:
+        if ape_code == APE_CODE[i][0]:
+            return i
+    return None
+
+def ape_list(ape_code):
+    import re
+    ape = list()
+    for ap in ape_code.split(","):
+        for i in APE_CODE:
+            if bool(re.match(ap,APE_CODE[i][0])):
```

Comment:
match return None in absence of match, or the match object on contrary. Therefore the bool casting is useless:
```
import re
if re.match(".*", "toto"):
...     print("I am in")
...     
I am in
if re.match("oups", "toto"):
...     print("I am in")
```

Reply:
Okay, i would like to be sure